e2e/runner/evm.go

```diff
@@ -181,6 +181,38 @@ func (r *E2ERunner) ApproveERC20(allowed ethcommon.Address) {
 	}
 }
 
+// ApproveETHZRC20 approves ETH ZRC20 on EVM to a specific address
+// check if allowance is zero before calling this method
+// allow a high amount to avoid multiple approvals
+func (r *E2ERunner) ApproveETHZRC20(allowed ethcommon.Address) {
+	allowance, err := r.ETHZRC20.Allowance(&bind.CallOpts{}, r.Account.EVMAddress(), r.GatewayEVMAddr)
```

Comment:
Some contract like USDT requires having allowance 0 to set a new one, therefore, moving forward it will be simpler setting a high allowance once rather than resetting the allowance